frontend/apps/tequila-train/src/components/sections/start-section/find-game.tsx

```diff
@@ -58,7 +59,7 @@ export const FindGame = ({ closeFindGame }: { closeFindGame: () => void }) => {
 
 	const onFindGame = () => {
 		setPreviousGame(null)
-		const findGame = state?.games.find(game => game[0] === form.values.creator)
+		const findGame = state?.games.find(game => game[0] === form.values.creator || game[0] === decodeAddress(form.values.creator))
```

Comment:
Are you sure that `creator` address there could be decoded and encoded at the same time?